### PR TITLE
Adds soy cilk

### PR DESCRIPTION
**Pull request by N3D6**
# Document the changes in your pull request

Adds soy cilk, just cilk but with soy milk instead of normal milk. Similar to how bilk has both soy and normal

# Wiki Documentation

Drinks page, add soy cilk

# Changelog

:cl:  
rscadd: adds soy cilk and the recipe to produce it
/:cl:

### CODE REVIEW

**Comment by Anvilman6** [comment]
image

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -57,7 +57,7 @@
 	glass_icon_state = "carrotjuice"
 	glass_name = "glass of  carrot juice"
 	glass_desc = "It's just like a carrot but without crunching."
-
+m
```
Comment (patpol4) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: m

**Comment by patpol4** [comment]
Also, bloat imo

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 1 above)
Comment (N3D6) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Fuck

**Comment by N3D6** [comment]
> Also, bloat imo

There's already a milk based drink called bilk that was added before cilk, it has a soy option as well. Better for consistency

**Comment by Tipy1802** [comment]
N3D6 is a soy boy

**Comment by JamieD1** [comment]
No. Bloat. Stupid. We let you have the first one.

**Pull request closed without merging**